babel.config.js

```diff
@@ -1,12 +1,12 @@
 module.exports = api => {
-  api.cache(false);
+  const env = api.env();
 
   return {
     presets: [
       [
         '@babel/preset-env',
         {
-          modules: false,
+          modules: env === 'test' ? 'commonjs' : false,
```

Comment:
* Issue : https://stackoverflow.com/questions/49396736/can-webpack-4-modules-be-configured-as-to-allow-jasmine-to-spy-on-their-members/52212382